porcupine/plugins/highlight/tree-sitter-token-mappings/python.yml

```diff
@@ -24,7 +24,29 @@ queries:
       return_type: (type)? @recurse
       (":") @Token.Punctuation
       (comment)? @Token.Comment
-      body: (block) @recurse)
+      body: (_) @recurse
+    )
+
+  decorated_definition: |
```

Comment:
Is there any reason to put this between `function_definition` and `class_definition`? I would prefer it to be either before both or after both, because `function_definition` and `class_definition` are very similar.